template/scripts/sample-script.js

```diff
@@ -13,13 +13,21 @@ async function main() {
   // manually to make sure everything is compiled
   // await hre.run('compile');
 
-  // We get the contract to deploy
-  const Greeter = await hre.ethers.getContractFactory("Greeter");
-  const greeter = await Greeter.deploy("Hello, Hardhat!");
+  // We get the contracts to deploy
+  const ERC20Token = await hre.ethers.getContractFactory('ERC20Token');
```

Comment:
ERC20 Token kinda clash with the name for `ERC20`. Do we want some names that relates to the DAO? something like `DevToken` :)

Reply:
If this is to be generic, `ERC20Token` seems reasonable for the contract, but implementation we could do something like `DevToken`.  It makes sense to us as DevDAO, but for other DAOs not so much.

Reply:
Yeah we want the name to be generic. Not sure what a good name is for that use case.